Author: Dererk <dererk@debian.org>
Description: Just use random servers from the Debian public NTP Pool 

diff --git a/ntpd.conf b/ntpd.conf
index afe5223..b230e60 100644
--- a/ntpd.conf
+++ b/ntpd.conf
@@ -11,4 +11,10 @@
 
 # use a random selection of 8 public stratum 2 servers
 # see http://twiki.ntp.org/bin/view/Servers/NTPPoolServers
-servers pool.ntp.org
+#servers pool.ntp.org
+
+# Choose servers announced from Debian NTP Pool
+servers 0.debian.pool.ntp.org
+servers 1.debian.pool.ntp.org
+servers 2.debian.pool.ntp.org
+servers 3.debian.pool.ntp.org
